src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  static updateAlive() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
```

Comment:
The updateAlive method currently uses filter which is good for performance as it avoids nested loops. However, make sure to call this method only when necessary to avoid unnecessary iterations over the Animal.alive array.